srcpkgs/alot/template

```diff
@@ -1,16 +1,25 @@
 # Template file for 'alot'
 pkgname=alot
-version=0.8.1
-revision=2
+version=0.9
+revision=1
 archs=noarch
 build_style=python3-module
-pycompile_module="alot"
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-Sphinx"
 depends="python3-setuptools python3-Twisted notmuch-python3 python3-configobj
  python3-gpg python3-magic python3-urwidtrees"
+checkdepends="$depends"
```

Comment:
you don't seem to be building from the same tree as i am,
adding `checkdepends="$depends gpgme procps-ng"` fixes the 3 errors

Reply:
I guess its using my own system then, where both dependencies are already available. How do you execute tests within the builddir?

Reply:
I think  `gpg` or `gpgme` should maybe be added as optional dependencies, because `alot` can function without?

Reply:
Nevermind, it's `checkdepends` not depends. I'll fix it.

Reply:
Ok, I've pushed your suggested changes now (and I've learned `./xbps-src check` is a thing, instead of `cd`'ing into the build dir... :facepalm:).

Reply:
@rien: set `XBPS_CHECK_PKGS=yes` in `etc/conf`

Reply:
xbps doesn't have the concept of optional dependencies, so I suggest to omit gpg stuff in `depends` but add them to `checkdepends`.
as pyhon3-pgp is already in `depends` (which pulls libgpgme), you better add gpgme to `depends` as well or remove all of them from it if. I vote for keeping gpg

Reply:
I agree in keeping the `python3-gpg`  dependency (because alot will probably crash if it isn't), but I would not explicitly add `gpg` or `gpgme` as dependency?

Reply:
as you see in the failed test, python3-gpg doesn't work when gpgme is missing

Reply:
i suggest to just add gpgme to depends as well

Reply:
`gpgme` wasn't installed on my own system while alot was working fine, I experimented a bit and `gnupg2` also works as a crypto engine. This is what I have installed, so I will add that as a hard dependency if that's OK for you.

Reply:
IIUC, either `gpgme` or `gnupg2` in `depends` is fine.
And `alot` doesn't require either packages.
It's only used when it see a mail is signed/encrypted.

I think I will add neither to `$depends`.
Just like `mutt`.

Reply:
I agree with @sgn so I've moved `gnupg2` to the `checkdepends` again. I have tested it in an empty docker and can confirm alot will start without  `gnupg2`  or `gpgme`, but it won't open signed emails. The error is concise: `GPGME: Invalid crypto engine`, so most users will know what's going on and install a gpg provider. I can add a post-install message if you so desire, but I don't think that's really necessary.

I've been going _a lot_ of back and forth here, and I think we're getting into bike shed territory here.

Reply:
This package is `alot` :D 

>     * 9 tests are failing, but they are 'expected'  to fail (`OK (expected failures=9)`)
> 
>     * I noticed that I couldn't reply because of an error, so I've also applied [this](https://github.com/pazz/alot/commit/28a4296c7f556c251d71d9502681980d46d9fa55) patch.
> 
>     * `$makejobs` was added.

My latest `check result` is:

> FAILED (failures=1, expected failures=9)